_docs/ideal_indirection.md

```diff
@@ -14,32 +14,52 @@ wikibook:
 
 ## Overview
 
-In this assignment, you will be working on managing a mapping of 32-bit virtual address space  to a 32-bit physical address space with 4 KB (kilobyte) pages.
+In this assignment, you will be working on managing a mapping of a 32-bit virtual address space to a 32-bit physical address space with 4 KB (kilobyte) pages.
 Each component of virtual memory has been split up into several files to help you build a mental model of virtual memory.
-Reading through these files can start to help you understand the roles of the different hardware and software involved in managing virtual memory.
-You will only have to write two functions in `mmu.c`, but it requires a good understanding of the coursebook and decent knowledge of the rest of the provided code.
+Reading through these files will help you understand the roles of the different hardware and software involved in managing virtual memory.
+You will only have to write two functions in `mmu.c`, but it requires a good understanding of the virtual memory translation process and decent understanding of the rest of the provided code.
+
+## How To Tackle This Assignment
```

Comment:
+1 - thanks for adding :)

Reply:
You know an assignment is special when that is added